upload-plugin/src/main/groovy/ly/count/android/plugins/uploadSymbols.groovy

```diff
@@ -58,27 +55,17 @@ class UploadSymbolsPlugin implements Plugin<Project> {
             .build()
         request = new Request.Builder().url(url).post(formBody).build()
       }
+      logger.debug("uploadJavaSymbols, Generated request: {}", request.body().toString())

```

Comment:
u sure?

Reply:
ups, why this is here and seems like a new thing

Reply:
sumimasee, this pr become a godzilla